spec/models/accounting/quickbooks/updater_spec.rb

```diff
@@ -35,5 +44,73 @@
         expect { subject.update }.not_to raise_error
       end
     end
+
+    context 'with new JournalEntry' do
+      before do
+        allow(subject).to receive(:changes).and_return('JournalEntry' => [journal_entry])
+      end
+
+      let(:last_updated_at) { 30.days.ago }
+      let(:qb_id) { 34 }
+      let(:journal_entry) { instance_double(Quickbooks::Model::JournalEntry, id: qb_id) }
+
+      subject { described_class.new(connection) }
+
+      it 'creates a new transaction with the proper qb_id' do
+        expect { subject.update }.to change { Accounting::Transaction.where(qb_id: qb_id).count }.by 1
```

Comment:
Since we're using DB cleaner there should only be one transaction in the DB, right? So if you do `take` as below and get something then you know it was created with correct ID